src/asm/fixpoint.c

```diff
@@ -19,17 +19,23 @@
 #include "asm/symbol.h"
 #include "asm/warning.h"
 
-#define fix2double(i)	((double)((i) / 65536.0))
-#define double2fix(d)	((int32_t)round((d) * 65536.0))
-
-// pi radians == 32768 fixed-point "degrees"
-#define fdeg2rad(f)	((f) * (M_PI / 32768.0))
-#define rad2fdeg(r)	((r) * (32768.0 / M_PI))
-
 #ifndef M_PI
 #define M_PI 3.14159265358979323846
 #endif
 
+#ifndef M_TAU
+#define M_TAU (M_PI * 2)
```

Comment:
That seems redundant and confusing if you don't know about tau (which is less common than π)

Reply:
Okay.